### PR TITLE
Propose Transfer Hotspot

### PR DESCRIPTION
**Pull request by abhay**
* PR #42
* Author: @mrallen1, @abhay 
* HIP Number: TBD
* Tracking issue: TBD

**Summary:** This proposal introduces a new type of transaction to the Helium blockchain called transfer_gateway_v1. This transaction will be an atomic transfer of an onboarded Hotspot / gateway from one owner to another. This transaction can have an optional amount of HNT associated.

**Rendered version:** https://github.com/helium/HIP/blob/9cf52310b532d98232d21de33449ce789c58ef10/0099-transfer-hotspot.md

### CODE REVIEW

File: 0099-transfer-hotspot.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+1. Current owner creates a partially signed transaction with an optional HNT amount that is required to complete the transaction.
+2. Current owner sends the partially signed transaction to the receiving owner.
+3. Recipient signs the transaction and pays the DC fee to submit the transaction to the blockchain.
+4. If the receiving account contains sufficient HNT balance as requested by the current owner and contains enough HNT to burn into DCs for the transaction, the transaction is accepted and the gateway's owner is updated in the ledger.
+5. The hotspot no longer appears in the sender's hotspot list and now appears in the recipient's hotspot list.
+
+## Implement the transaction in the helium-wallet client
+
+`TODO`
```
Comment (mrallen1) [inline 0099-transfer-hotspot.md:44]: @madninja and I talked about this today and we're not sure we should go into great detail here - the helium-wallet already supports "exporting" a partially completed transaction as either a QR code format or a Base64 encoded text blob - either of these formats would be useful for completing the transaction.

Comment (abhay) [reply 0099-transfer-hotspot.md:44]: That's probably enough information exactly as described. Want to add it or me?

Comment (abhay) [inline 0099-transfer-hotspot.md:44]:
```suggestion
The [Helium Wallet CLI](https://github.com/helium/helium-wallet-rs) currently supports exporting a partially completed transaction as either a QR code or a Base64-encoded text blob. Either of these formats would be useful for competing the transaction. 
```

File: 0099-transfer-hotspot.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# HIPxx: Transfer Hotspot
+
+- Start Date: 2020-09-22
+- Original HIP PR: [#42](https://github.com/helium/HIP/pull/42)
+- Author: [@abhay](https://github.com/abhay)
```
Comment (abhay) [inline 0099-transfer-hotspot.md:5]:
```suggestion
- Author: [@mrallen1](https://github.com/mrallen1), [@abhay](https://github.com/abhay)
```

**Comment by abhay** [comment]
@jamiew I feel like this is good to land. Should I adjust the HIP # and filename?

File: 0099-transfer-hotspot.md (hunk 1 above)
Comment (mrallen1) [reply 0099-transfer-hotspot.md]: Sure thanks

**Comment by jamiew** [comment]
Numbered, squashed & merged. Thanks for your business!

Please direct future discussion to the tracking issue: https://github.com/helium/HIP/issues/43

**Pull request merged**